changelog.d/5843.misc

```diff
@@ -0,0 +1 @@
+Formatting XML resources to a create baseline in preparation for future changes
```

Comment:
```suggestion
Formatting XML resources to create a baseline in preparation for future changes
```

(I may be wrong, you are the native English speaker :) )

Reply:
:facepalm: one day I'll learn English....